assets/js/handlebars/templates/autocomplete-suggestions.handlebars

```diff
@@ -6,7 +6,7 @@
         Autocompletion results for <span class="bold">"{{term}}"</span>
       </span>
       <span class="press-return">
-        Press <span class="bold">RETURN</span> for full-text search, <span class="bold">TAB</span> for previews
+        Press <span class="bold">RETURN</span> for full-text search, <span class="bold">→</span> for previews
```

Comment:
I am sorry but I think we should go with TAB, as someone can still use the arrows to change the input itself. Can we revert to tab?

Reply:
@josevalim ok, understood

Reply:
@josevalim Q: how to close preview by keyboard? Is it good idea using  `shift + Tab` ?

Reply:
Generally speaking, I think the current navigation looks good, so unless you find a bug (code bug or UI bug), I'd be happy to keep it as is. :)

Reply:
@ThaddeusJiang it should just be pressing Tab again. It should work right now as well!

Reply:
@josevalim understood

As my test, the TAB key was broken.

before:
![CleanShot 2024-04-08 at 15 47 54](https://github.com/elixir-lang/ex_doc/assets/17308201/0de73bb8-27fa-4026-a66c-e807dd527f56)

So, I create this PR.

after: 

![CleanShot 2024-04-08 at 15 49 34](https://github.com/elixir-lang/ex_doc/assets/17308201/26a82446-3a30-4fe9-b2a7-0166af31d042)

Reply:
@ThaddeusJiang interesting. which browser are you using?

Reply:
@josevalim Arc, I am using https://arc.net/

Reply:
@ThaddeusJiang this is weird. I have tested on both Arc and Safari  and they both work just fine. Have you run `mix build` before testing it, just in case?

Reply:
I have also noticed that your preview shows the search bar and it does not show up for me. :(

Reply:
That's it, great find. It should be "open doc/index.html". Maybe we should document it somewhere?

Reply:
I tried to improve the README instructions for clarity. Btw, do you think we need something from this PR now? Thanks!

Reply:
@josevalim I have fixed the click issue in this PR.
Could you review it?
 https://github.com/elixir-lang/ex_doc/pull/1887/files